src/code/game.c

```diff
@@ -58,7 +58,7 @@ void GameState_SetFBFilter(Gfx** gfx) {
     *gfx = gfxP;
 }
 
-void func_800C4344(GameState* gameState) {
+void GameState_UpdateRegs(GameState* gameState) {
```

Comment:
UpdateRegs doesn't really capture the purpose of this function, though I imagine it's hard to give a good name for this function since it performs various seemingly unrelated data logging tasks and then also updates the RegEditor.

Reply:
Yeah this function seems to do various debug things, including logging stuff and setting the log verbosity on a bunch of systems, so maybe it should be named around that (eg. `GameState_DebugUpdate`)

Reply:
I reverted the name back to `func_800C4344`, there are other debug functions to name, may as well do them later